### ISSUE TITLE
Update to Font Awesome 5

### ISSUE THREAD

**Issue by antonjarl** [home-assistant/appdaemon#208]
A couple of weeks ago Font Awesome was updated to version 5, which includes much more icons, and also different version of all icons: Line, Regular and Solid.

The new icons looks way better and way more consistent than the old ones. Upgrading to FA5 would  be much appreciated!

**Comment by acockburn** [comment]
Yep - I was also looking at upgrading the MDI icons, so I'll do both in 3.0

**Comment by antonjarl** [comment]
Awesome @acockburn. Just ping me if you need help testing it out.

**Comment by acockburn** [comment]
I will do this but I have decided to defer it until after the beta - upgrading will create a lot of breaking changes to peoples dashboards as a lot of icon names have changed, so I'll slip this in when things have settled down a bit.

**Comment by ID4600** [comment]
Hi, I can't find a topic that completely matches my problem, but I can't get fa and mdi icon's to work . 
They do work for the lights and switches with an on and off state but I can't get them to work on the static buttons. I use `icon: fa-home ` for declaring the icon, I've tried  `icon_on` , `icon_off`and so on, but no response just the generic gear. 

What am I doing wrong? 

I'm running HA 0.64.0
AppDaemon3 0.1.0

**Comment by acockburn** [comment]
Hi there - please ask this in the forum as it's likely a config issue.

**Comment by jcollie** [comment]
Would sure be nice to get this updated. Is there a way to include both, and be able to choose? Maybe the "fa-" prefix uses the v4 icons and the "fa5-" prefix uses the v5 icons?

**Comment by acockburn** [comment]
Unfortunately I don;t think that can be done due to collisions in the stylesheet naming - it would probably require the user to make a decision for each dashboard and the code to do that would need to be added.

**Comment by jcollie** [comment]
Now that I think about it some, I don't think that mixing v4 and v5 icons on the same dashboard is necessary, or even on different dashboards. Having a global configuration option that would switch between v4 and v5 on all dashboards would be enough for me.

**Comment by antonjarl** [comment]
Actually the v5 icons uses other prefixes: fal, fas, far for line, regular and solid and fab for brands. And Font Awesome provides way to upgrade it smoothly. I wouldn't recommend trying to make both live side by side , it'll be a lot of problems.

**Comment by acockburn** [comment]
This will be in v3.02 which will be out this week or next.

**Comment by antonjarl** [comment]
Nice!
